proto/zetachain/zetacore/crosschain/cross_chain_tx.proto

```diff
@@ -54,28 +54,28 @@ message ZetaAccounting {
   ];
 }
 
-message OutboundTxParams {
+message OutboundParams {
```

Comment:
This would impact (or break?) the indexer/explorer/hub, full-stack team also need to prepare for the structure changes.

Reply:
Yes, it might but I'm planning a revamp of the CCTX object in general for v17 so it will be easier to set all the changes once we are at this point